src/esmockModule.js

```diff
@@ -92,14 +97,24 @@ const esmockModuleCreate = async (treeid, def, id, fileURL, opt) => {
   return mockModuleKey
 }
 
+const esmockMetaResolve = async (id, parent) => {
+  try {
+    return decodeURI(await import.meta.resolve(id, asFileURL(parent)))
```

Comment:
hmm... here we can also use node's function, right?
https://nodejs.org/dist/latest-v19.x/docs/api/url.html#urlfileurltopathurl

Reply:
This suggestion is correct BUT the change fails the tests now, because the other esmock parts need decoded fileURL. I think the best overall solution is to update resolvewithplus to return encodedURI, just like import.meta.resolve, then update esmock sources to use the encoded version of the url directly. If its okay with you, these could be done in follow-up PR

Reply:
Yep, agree. Thx!

ps added comment about decode / encode in another PR, but looks like it will be refactored anyway.